### ISSUE TITLE
Tooltip on stacked chart list values in reverse order to the chart stacks.

### ISSUE THREAD

**Issue by prestond123** [FormidableLabs/victory#666]
This this is counter intuitive.
Here is a fiddle.  https://jsfiddle.net/3zn2o67d/

**Comment by jadus** [comment]
I have this issue too. Found a workaround ?

**Comment by prestond123** [comment]
Sorry no, I didn't find a workaround.  Still waiting for a fix, this is low priority for us, so not been chasing - as the datasets are colour coded in the tips, so It's relatively obvious what the tip is conveying.

**Comment by jadus** [comment]
How can you color code the tooltip values ?
Thanks.

**Comment by prestond123** [comment]
I may have to do some digging - to remember that,  but at a glance it was  like this:
_(colourArray holds my dataset colours)_

```
const styleArr = colourArray.map(ca => ({ fill: ca }) );
<VictoryTooltip style={styleArr}  />
```

but I also have a style on the Area (but I dont think it was this)
```
<VictoryArea
    style={{
                    data: { stroke: colourArray[i] },
                    labels: { fill: colourArray[i] }
              }}
              data={data}/>
```

**Comment by jadus** [comment]
Thanks a lot, I'll try this

**Comment by prestond123** [comment]
Post back what worked - if you can.  Thanks

**Comment by stefvhuynh** [comment]
It looks like this behavior has changed in the latest versions of `victory`. When I update the `victory` dependency in the fiddle to the most recent one, the tooltip is no longer stacked. It only shows one number a time now. @prestond123, is this still an issue?

**Comment by stefvhuynh** [comment]
i was able to reproduce the issue by setting `voronoiDimension="x"`.

**Comment by boygirl** [comment]
This issue is fixed in the latest version `victory@0.27.1`
